presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -153,3 +153,8 @@ Hive, this table can be described in Presto::
 This table can then be queried in Presto::
 
     SELECT * FROM hive.web.page_view;
+
+Hive Connector Limitations
```

Comment:
This section can just be "Limitations" since this is already the Hive connector docs